_episodes/07-resources.md

```diff
@@ -117,20 +117,22 @@ Rules claiming more threads will be scaled down.
 
 Our pipeline ran in parallel and finished roughly 4 times as quickly! The
 takeaway here is that all we need to do to scale from a serial pipeline is
-run `snakemake` with the `-j` option. By analysing the dependencies between
+run `snakemake` with the `-c 4` option. By analysing the dependencies between
 rules, Snakemake automatically identifies which tasks can run at the same
 time. All you need to do is describe your workflow and Snakemake does the
 rest.
 
-Note you can also use `snakemake --cores 4` or `snakemake --jobs 4`. The
-`-j`, `--cores` and `--jobs` arguments all mean the same thing.
+Note you can also use `snakemake --cores all` or `snakemake --jobs all`. The
+`-c`, `-j`, `--cores` and `--jobs` arguments all mean the same thing
+when running locally on your own machine.
```

Comment:
Recommend to put this in a callout. I think it is not essential information to follow the lesson, and introducing two ways to achieve the same thing (but only in a certain context) gives little benefit but at the risk of confusing people.

Reply:
Ok.